lib/travis/api/app/endpoint/authorization.rb

```diff
@@ -288,9 +287,10 @@ def fetch
               if user
                 rename_repos_owner(user.login, info['login'])
                 user.update_attributes info
+                Travis.run_service(:sync_user, user) if user.previous_changes[:github_oauth_token]
               else
                 self.user = ::User.create! info
```

Comment:
I believe we do. I haven't changed line 292 (the assignment you've commented on), but added `self` to the next line 293 where it is supposed to pass the new/created user to the sync service. IIRC it didn't access `self.user` there because the local variable `user` already exists in the same scope.